### ISSUE TITLE
feat(log-viewer): ui to view bot/instance logs

### ISSUE THREAD

**Issue by CosmoAbdon** [baptisteArno/typebot.io#1706]
**Is your feature request related to a problem? Please describe.**
We need better ways to look for errors/console.
Example: Some users faced errors on BOT with API/Integrations usage.
What could be happening? To check that, the infra team, must go to the server and look for the logs (or cloudwatch logs)

**Describe the solution you'd like**

We can create an LogViewer, so we can always look for logs.
I thought about 2 ways of doing that.
1 - Workspace view, where we only view for workspace (the user is) + BOT Selector.
So we can look about the error in specific BOT and etc.
2 - Application view (For this we may need to create an user role like super_admin or a new frontend for that.), where we can select the workspace/bot (So we can impesonate)

We can have:
- Popup/Modal LogViewer
- Drawer (like google chrome console).

We can have things like: Botpress/Rancher logs.
Get the: 'last X lines' / 'last X minutes' PLUS live log.

This can be way to good for development and production errors.

I can help @ at backend level, but I think we need to discuss about how I can implement that.
So I can follow the ideia/concept of code,structure you have in mind for the future of the project.

I'm really that bad @ front end, so I may need your help or from someone at community.

**Comment by baptisteArno** [comment]
No, server logs should never be displayed on frontend. This is really not a good idea 🙏
